cisagov/getgov
# List users on homepage

## 🗣 Description ##

<img src="https://user-images.githubusercontent.com/15718530/202494888-142e7f90-24b0-48b9-ae4e-d369678005d7.png" width="500px" />

## 💭 Motivation and context ##

<strike>This PR conflicts with #277 in the use of `Website` vs `Domain` for the `requested_domain` field. This must be updated before merging.</strike> Will address on #277 

Closes #261 

## 🧪  Testing steps ##

1. Use the [admin](http://localhost:8080/admin/) to add a few domain applications, some with your user as a the creator and some with another user (`docker-compose exec app ./manage.py loaddata registrar/fixtures/*` if you have no other users).
2. Verify that all your applications show up and no others
3. Use the frontend testing checklist to verify no accessibility issues
